### PR TITLE
change rosgraph_manipulator_config.yaml path

### PR DESCRIPTION
**Pull request by Rezenders**
fix issue #59

### CODE REVIEW

File: launch/includes/rosgraph_nodes.launch.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <launch>
 
-  <rosparam ns="rosgraph_manipulator" command="load" file="$(find metacontrol_sim)/yaml/rosgraph_manipulator_config.yaml"/>
+  <rosparam ns="rosgraph_manipulator" command="load" file="$(find mc_rosgraph_manipulator)/config/rosgraph_manipulator_config.yaml">
```
Comment (marioney) [inline launch/includes/rosgraph_nodes.launch.xml:3]: Seems fine, just a question.

Why did you remove the `/` from the tag.  I'm not sure if this will work correctly that way

Comment (Rezenders) [reply launch/includes/rosgraph_nodes.launch.xml:3]: Ops, just a typo. I will correct it.

**Comment by Rezenders** [comment]
I messed up this PR so I had to open another one. https://github.com/rosin-project/metacontrol_sim/pull/62
